snapshot/git/gitdb/schemes.go

```diff
@@ -0,0 +1,65 @@
+package gitdb
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/scootdev/scoot/snapshot"
+)
+
+// GitDB uses several different Schemes to identify, upload and download Values.
```

Comment:
Value is just too generic, same as saying it stores Foo imo. If there's some concept that's more specific, we should find a word for that.